azure-pipelines.yml

```diff
@@ -218,6 +218,9 @@ stages:
               PathtoPublish: $(Build.ArtifactStagingDirectory)\tests\scenarios
               ArtifactName: ScenarioTests
               ArtifactType: Container
+          
+          - task: CopyFiles@2
```

Comment:
What files is this copying? It doesn't have any parameters.

Reply:
I... didn't mean to change this file at all. i'll revert it